app/src/main/java/com/cornellappdev/transit/ui/theme/Color.kt

```diff
@@ -8,4 +8,8 @@ val Pink80 = Color(0xFFEFB8C8)
 
 val Purple40 = Color(0xFF6650a4)
 val PurpleGrey40 = Color(0xFF625b71)
-val Pink40 = Color(0xFF7D5260)
\ No newline at end of file
+val Pink40 = Color(0xFF7D5260)
+
+val transitBlue = Color(0xff079ddc)
+val primaryText = Color(0xff212121)
+val secondaryText = Color(0xff616161)
```

Comment:
Could you name these with capitalized names (e.g. `TransitBlue`) to follow convention here?

Reply:
updated!